src/data_conversion.rs

```diff
@@ -592,816 +572,26 @@ pub fn convert_network_data_points(
-                    cpu_percent_usage: process.cpu_usage_percent,
-                    mem_percent_usage: process.mem_usage_percent,
-                    mem_usage_bytes: process.mem_usage_bytes,
-                    mem_usage_str,
-                    group_pids: vec![process.pid],
-                    read_per_sec,
-                    write_per_sec,
-                    total_read,
-                    total_write,
-                    rps_f64: process.read_bytes_per_sec as f64,
-                    wps_f64: process.write_bytes_per_sec as f64,
-                    tr_f64: process.total_read_bytes as f64,
-                    tw_f64: process.total_write_bytes as f64,
-                    process_state: process.process_state.to_owned(),
-                    process_char: process.process_state_char,
-                    process_description_prefix: None,
-                    is_disabled_entry: false,
-                    is_collapsed_entry: false,
-                    user,
-                };
-            }
-        } else {
-            existing_converted_process_data.insert(
-                process.pid,
-                ConvertedProcessData {
-                    pid: process.pid,
-                    ppid: process.parent_pid,
-                    is_thread: None,
-                    name: process.name.to_string(),
-                    command: process.command.to_string(),
-                    cpu_percent_usage: process.cpu_usage_percent,
-                    mem_percent_usage: process.mem_usage_percent,
-                    mem_usage_bytes: process.mem_usage_bytes,
-                    mem_usage_str,
-                    group_pids: vec![process.pid],
-                    read_per_sec,
-                    write_per_sec,
-                    total_read,
-                    total_write,
-                    rps_f64: process.read_bytes_per_sec as f64,
-                    wps_f64: process.write_bytes_per_sec as f64,
-                    tr_f64: process.total_read_bytes as f64,
-                    tw_f64: process.total_write_bytes as f64,
-                    process_state: process.process_state.to_owned(),
-                    process_char: process.process_state_char,
-                    process_description_prefix: None,
-                    is_disabled_entry: false,
-                    is_collapsed_entry: false,
-                    user,
-                },
-            );
-        }
-    }
-
-    // Now clean up any spare entries that weren't visited, to avoid clutter:
-    complete_pid_set.iter().for_each(|pid| {
-        existing_converted_process_data.remove(pid);
-    })
-}
-
-const BRANCH_ENDING: char = '└';
-const BRANCH_VERTICAL: char = '│';
-const BRANCH_SPLIT: char = '├';
-const BRANCH_HORIZONTAL: char = '─';
-
-pub fn tree_process_data(
-    filtered_process_data: &[ConvertedProcessData], is_using_command: bool,
-    sorting_type: &ProcessSorting, is_sort_descending: bool,
-) -> Vec<ConvertedProcessData> {
```

Comment:
All of this was removed and switched to being handled by `ProcWidget`. All the logic also follows the tuine rewrite logic.